deployment/cdk/opensearch-service-migration/lib/stack-composer.ts

```diff
@@ -179,22 +179,7 @@ export class StackComposer {
         const sourceClusterEndpoint = this.getContextForType('sourceClusterEndpoint', 'string', defaultValues, contextJSON)
         const osContainerServiceEnabled = this.getContextForType('osContainerServiceEnabled', 'boolean', defaultValues, contextJSON)
 
-        const migrationAnalyticsServiceEnabled = this.getContextForType('migrationAnalyticsServiceEnabled', 'boolean', defaultValues, contextJSON)
```

Comment:
We have an additional section in the `options.md`  file with these options, can you remove that specific section as well